### PR TITLE
chore(early-access-feature): remove scene and use table

### PR DESCRIPTION
**Pull request by EDsCODE**
- don't use persons scene in another scene
- update early access scene to use the person table directly with related components

### CODE REVIEW

File: frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx [hunk 1]
```diff
@@ -303,37 +305,57 @@ function PersonList({ earlyAccessFeature }: PersonListProps): JSX.Element {
...
     const logic = personsLogic(personsLogicProps)
-    const { persons } = useValues(logic)
+    const { persons, personsLoading, listFilters } = useValues(logic)
+    const { loadPersons, setListFilters } = useActions(logic)
     const { featureFlag } = useValues(featureFlagLogic({ id: earlyAccessFeature.feature_flag.id || 'link' }))
 
     return (
         <BindLogic logic={personsLogic} props={personsLogicProps}>
             <h3 className="text-xl font-semibold">Opted-In Users</h3>
-            <PersonsScene
```
Comment (neilkakkar) [inline frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx:312]: @Twixes - it's now annoying that we have to replicate all the logic from the scene (to add the filter & search on top). Is there another component we could use? Or does it make sense to make Search + Filter + List a component that's used inside the scene & here too?

Comment (EDsCODE) [reply frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx:312]: 👆 agreed, originally why I used the scene

Comment (EDsCODE) [reply frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx:312]: Can we get this in in the meantime?

Comment (neilkakkar) [reply frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx:312]: yes!

**Pull request merged**